src/commands/code-analyzer/config.ts

```diff
@@ -17,9 +17,13 @@ export default class ConfigCommand extends SfCommand<void> implements Displayabl
 	public static readonly summary = getMessage(BundleName.ConfigCommand, 'command.summary');
 	public static readonly description = getMessage(BundleName.ConfigCommand, 'command.description');
 	public static readonly examples = getMessages(BundleName.ConfigCommand, 'command.examples');
+
 	// TODO: UN-HIDE WHEN COMMAND IS READY
 	public static readonly hidden = true;
 
+	// TODO: Update when we go to Beta and when we go GA
+	public static readonly state = getMessage(BundleName.Shared, 'label.command-state');
```

Comment:
Is `state` a property that we're inheriting from upstream?

Reply:
The state property is on the Command (superclass of SfCommand) which automatically updates the help text for you with "This command is in %s.". It doesn't provide a warning... which is why I also added the same message in the message catalog for the warning. (see slack conversation)